readme.txt

```diff
@@ -86,6 +86,7 @@ Follow the steps below to connect the plugin to your PayPal account:
 * Fix - Multi-currency broke #481
 * Fix - Address information from PayPal shortcut flow not loaded #451
 * Fix - WooCommerce as mu-plugin is not detected as active #461
+* Fix - Check if PayPal Payments is an available gateway before displaying it on Product/Cart pages #447
```

Comment:
Why is it in this PR? :thinking:

Reply:
Not sure, but seems because it was not committed before I created this branch from trunk, in any case, I´ve checked it and it´s already in trunk: https://github.com/woocommerce/woocommerce-paypal-payments/blob/trunk/changelog.txt#L8

Reply:
Sorry my bad, it does not belong to this PR, removed.